packages/driver/src/cypress/error_utils.ts

```diff
@@ -572,6 +586,7 @@ export default {
   mergeErrProps,
   modifyErrMsg,
   processErr,
+  serializeCrossDomainErrorsFromObject,
```

Comment:
I'd like to somehow register this with the `communicator` in some way, so that we don't have to call it when Errors stream back in from the secondary. Any ideas?